packages/nimble-components/src/table-column/enum-text/tests/table-column-enum-text.spec.ts

```diff
@@ -0,0 +1,492 @@
+import { html, repeat } from '@microsoft/fast-element';
+import { Table, tableTag } from '../../../table';
+import { TableColumnEnumText, tableColumnEnumTextTag } from '..';
+import { waitForUpdatesAsync } from '../../../testing/async-helpers';
+import { type Fixture, fixture } from '../../../utilities/tests/fixture';
+import type { TableRecord } from '../../../table/types';
+import { TablePageObject } from '../../../table/testing/table.pageobject';
+import { wackyStrings } from '../../../utilities/tests/wacky-strings';
+import { getSpecTypeByNamedList } from '../../../utilities/tests/parameterized';
+import { MappingText, mappingTextTag } from '../../../mapping/text';
+import { mappingSpinnerTag } from '../../../mapping/spinner';
+import { mappingIconTag } from '../../../mapping/icon';
+
+interface SimpleTableRecord extends TableRecord {
+    field1?: string | number | boolean | null;
+    field2?: string | number | boolean | null;
+}
+
+interface BasicTextMapping {
+    key?: string | number | boolean;
+    label: string;
+    defaultMapping?: boolean;
+}
+
+describe('TableColumnEnumText', () => {
+    let element: Table<SimpleTableRecord>;
+    let connect: () => Promise<void>;
+    let disconnect: () => Promise<void>;
+    let pageObject: TablePageObject<SimpleTableRecord>;
+
+    // prettier-ignore
+    async function setup(mappings: BasicTextMapping[], keyType = 'string'): Promise<Fixture<Table<SimpleTableRecord>>> {
+        return fixture<Table<SimpleTableRecord>>(
+            html`<${tableTag} style="width: 700px">
+                    <${tableColumnEnumTextTag} field-name="field1" key-type="${keyType}">
+                        Column 1
+                        ${repeat(() => mappings, html<BasicTextMapping>`
+                            <${mappingTextTag}
+                                key="${x => x.key}"
+                                label="${x => x.label}"
+                                ?default-mapping="${x => x.defaultMapping}">
+                            </${mappingTextTag}>
+                        `)}
+                    </${tableColumnEnumTextTag}>
+                    <${tableColumnEnumTextTag}>
+                        Column 2
+                    </${tableColumnEnumTextTag}>
+                </${tableTag}>`
+        );
+    }
+
+    afterEach(async () => {
+        await disconnect();
+    });
+
+    for (const test of [
```

Comment:
This should use the `getSpecTypeByNamedList` helper.

Reply:
Fixed